.github/workflows/ci.yml

```diff
@@ -0,0 +1,99 @@
+name: Unit tests
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    name: Python ${{ matrix.python-version }}, django ${{ matrix.django-version }}
+    runs-on: ubuntu-18.04
```

Comment:
I think this might be better as ` ubuntu-latest`

Reply:
Version 3.4 with arch x64 not found with latest ubuntu.
https://github.com/actions/setup-python/issues/185